apps/web/components/Cookiebar/Cookiebar.vue

```diff
@@ -1,24 +1,24 @@
 <template>
   <div
     v-if="visible"
-    class="fixed max-sm:flex max-sm:flex-col max-sm:justify-between z-50 w-full xl:w-3/5 xl:right-2 bottom-0 max-md:bottom-[3.9rem] shadow-2xl p-3 bg-white rounded overflow-auto top-0 sm:top-auto"
+    class="fixed cookieBarHeightForMobile max-sm:flex max-sm:flex-col max-sm:justify-between z-50 w-full xl:w-3/5 xl:right-2 bottom-0 shadow-2xl p-3 bg-white rounded overflow-auto end-0 sm:top-auto"
```

Comment:
1. Putting a fixed height on the whole cookie bar will cause double scroll. We don't need fixed height on the whole cookie bar.

<img width="339" alt="Screenshot 2024-10-01 at 15 45 45" src="https://github.com/user-attachments/assets/bdce95c6-e244-4476-ae05-f30c199652cf">

2. Removing `max-md:bottom-[3.9rem]` breaks a functionality we implemented a few days ago for PSI, please don't remove code if you are not 100% sure.

Reply:
Thank you for your time
1.From task description this is what i understood.  (maybee you can provide more context (pm)
2.(max-md:bottom-[3.9rem) was not removed accidentaly(maybee you can provide more context (pm)